packages/backend-deployer/src/cdk_deployer.ts

```diff
@@ -33,8 +33,14 @@ export class CDKDeployer implements BackendDeployer {
    */
   constructor(
     private readonly cdkErrorMapper: CdkErrorMapper,
-    private readonly backendLocator: BackendLocator
-  ) {}
+    private readonly backendLocator: BackendLocator,
+    private readonly packageManager: string
+  ) {
+    // TODO: use PackageManagerController to get the executable.
```

Comment:
I wonder if it's going to be `get the executable` or perhaps `packageManagerController.executeWithPackageManager(command, args)`.
e.g. `packageManagerController.executeWithPackageManager('tsc', tscArgs)`;

Reply:
That sounds a good idea 👍